ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-nightly.yaml

```diff
@@ -703,6 +703,44 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-heterogeneous
     workflow: cucushift-installer-rehearse-gcp-ipi
+- as: gcp-ipi-f28-compliance
+  cron: 33 11 25 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      COMPUTE_NODE_TYPE: t2a-standard-4
```

Comment:
You also need to add FILTERS_ADDITIONAL: "". Otherwise the test cases with NonPreRelease tag has no chance to run.